app/test_main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.has_internet_connection")
+@mock.patch("app.main.valid_google_url")
+def test_can_access_google_page(
+        mock_valid_google_url: mock.MagicMock,
+        mock_has_internet_connection: mock.MagicMock
+) -> None:
+    mock_has_internet_connection.return_value = True
+    mock_valid_google_url.return_value = True
```

Comment:
What about other cases, if this functions returns True/False False/True False/False